packages/app/src/app/pages/Sandbox/Editor/Workspace/items/GitHub/Git/Git.tsx

```diff
@@ -0,0 +1,156 @@
+import Margin from '@codesandbox/common/lib/components/spacing/Margin';
+import { Button } from '@codesandbox/common/lib/components/Button';
+import GithubBadge from '@codesandbox/common/lib/components/GithubBadge';
+import Input, { TextArea } from '@codesandbox/common/lib/components/Input';
+import Notice from '@codesandbox/common/lib/components/Notice';
+import { githubRepoUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { observer } from 'mobx-react-lite';
+import React, { useEffect } from 'react';
+
+import { useSignals, useStore } from 'app/store';
+
+import { WorkspaceSubtitle, WorkspaceInputContainer } from '../../../elements';
+
+import { Container, Buttons, ErrorMessage, NoChanges } from './elements';
+import { TotalChanges } from './TotalChanges';
+
+function hasWriteAccess(rights: 'none' | 'read' | 'write' | 'admin') {
+  return rights === 'write' || rights === 'admin';
```

Comment:
@Saeris You can just use `Array.includes()` here

```suggestion
  return ['admin', 'write'].includes(rights);
```